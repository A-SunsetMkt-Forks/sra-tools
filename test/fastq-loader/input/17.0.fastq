@0f950b71-118b-4c7d-8030-4610650687c2_Basecall_1D_template
UGCCCUGAUUCAUGAUCCCCGACGAUGGGGGGGCAAUUUUGAGGAGUUCCAUCUGUGGUUCCUACAAGAACUCGGGCCUUGAAACCUGGCCUAUGGCACUUCGAGAAGCAUAUCCUAUCUCAAGAACACAGUUUCCCCAUUUUGUGUGUGGUUCAAGUUCUGGUCAGUGAGAAACUCCAUGACCACAAAGUUUACCACGUCGGUAGUCAACUCCUUAAGUGGGAUCUUCCUUUAUGAUGAAACCAUACAAGUAUCGGUUUUCGCCUCACCCUCUUAGUUCUACUUGCUUAUCAUUGGAAUUAAAUUGGAAAUCAUCCAUUGCUGCACAAGAUGCUACUAAGUAAGGCAGUAUUGAUGUACACCCCUUAUGUAUUUUGUGGGCUCUGCAGUGAGACACCUCUGAUGUAAUUCCUCAUGCUUGCAGCGUGUCAAUUGGAGCCACAUCUCCAAUAAUAUCCAGCUUGAAAUUUGGGUCAGUUCGCAUGCACCAUUGUUGUUGAACUCAUUCUGAAUCCAUUCUUGCAAGCGACCUCAAUUCUGGUCUCAACCCAUUAUAUUGCUUCAAUUCUCCUAUUCAUCAGUCGUUUAAGUCUGGUGCCAUCGUUCUUAAGUGCCCACUUUGGUAUCCGUUCCACUCCCCCCUUUAGUUUUGGAAUUUUUCCUUCAUUUUUGUAUCCUGCAGUUUGCCAGUACUUGUUCAUGACAGAAGAUAAUUUGAUUUAUUCCCUUCUAUGGUUUAACAACAUUGGGUUCCAUCCAAAGAAUGUUCAUGCAUUUGAUUGCAUCAUAGCAGUAUUCCCUCUCCCUCCACAUGACUUUGGUCCUCAAUGCUUAAUUUUUGGGCAUCCAUCAGCAGGGAAUUUGGUCGCUGAGAACAGGAGGCCCAUGGAAGUCUAAGUGGUCUGGUGACUGCUUUCAAAAAGGUCGCUAGCAUUUACUUUGGACAUUUGAGACAGCUUGCCCCUUGUAGCUGCUUCGGUUCGAAUCCAUCCAUGGGCUCAUAAUACAAAGGCUAGAAAGUCUCGGCGAGACUUUGGUCGGCAAGCUUGCGCAUUGUUCCUGUGAUCUCAAACCUUCAAUAUUGUUCUGAAGACAAAGAUGAAAGAAAUACAAGUUGUUGGCCAUGGUUACGGUGAAUAGCCAAAGUUAAGAUCUAGCCAAAGUUCCUAUGAGAGUGAGUAGCCUGUGGGCCAGUGAACGAGAAAAAUGUGGAUGUGUGUGUUUCAGAUUUAAAUUUUAUUAUGGCCUUUCCCAGAUGGUAUGUGAACUUCCUGUUACUCCAAUUUCGAUGAAUCUAUUUUCCUGUAAUAUACAAAUCUGGUAGAAACUUGUUUCAGCCCAGCAGUGUUGCAAAAUUGGCACUACUAGUCCAGGCCAUUGUGCGAUCUUCCCAUGAUUAUUCAAAUCUGUGUGCUUUCAAAGUGCUAUUAGGAUCACCAAGUUCUACGAUUAUUGACUCGCCUUAGCUCUAUUGAUGAAGUAGAAUCUGAAUACUGUAGUAUUUUCUAGUGGAGUGCUAUUGCUGCAAAUUUGUUUUUCGAUUUUCAGGUCCCAUACUUUAUUGUUUUUUUUCAAGUCGACAAUUAUCGGAUUGAAGCAUUGUUGCAGCUACUUUGAUUUGAUCUGCUUUUGCUAUCUCCCAUCCCAUCCCGUUUUAUCCAUAUCAUCAACCAUCCCAUCCCUAUCUUCCUCAUAU
+
#"""######""##"###%##""#"####"#$%"$%'+20%$'(#,)+.+%%"(%'*-0./+*%%&&$)*'%%+42*.&,)3<04:(**+**((&5&'%&)(&/4740.)((%()'&)*$&&0(.5021*+,(&$+1.+*661.-,*,+/,-(+6+)++'*,+**)2,*)().2+,11..-%(**+//)-+.+&('&'&&'(,'$-'&(.'+)0)1*-+++34,(%(''&*0-*+036/=>9060.))()(('($+4)+-(%()+%'$'03$%*-)/*-$((''))3;60.)(+-,+.-,,.4+1/-./6*)&--1-+1./5885/-12.12/()+*0)/(&%&*+,,+,)*.)*--*(%#&063&((++03,0++(*-72&&#%""&**,+++,-7--).-+*))+.1)*)%%$%)%+',('*'***.+,1902/+-01''&&%)**0+%%3&(*513-*01-,2..5./+,(%&.+,)*'(***),*&'*().&&0-/,))*(-/)-3:320)-''%/51/1/-++),.-/7.-*&/+1*()'(*'(&'&+''%(+-),-%'0(+$'%)%&&)$$'$%%&)).&%.-)%%%#$,,,.''$$&'$(&(-(&(*420),6/)-(#%('''*'*##''.693.#%(&(+*.-)+.)+11*)(,'+('(1/*$$(*+2201/.+2)..54+)(((0*)%%'$$'''(*00.--*3+845--&++-/)3())$&#$)))/+/,+3-++-../0+*-*('+-+4..78-,,0.-.13110,,*+-+)1--)$+0/01.0)*26,&&'*+*%)+&)++,/)*-,1+*4--14654++--4(,,,.(2,))+01/0,,+*)+7,4.0'//*&+))('-).1*%)*$'.*5'&&$&/1)*).----.++&)*,(*(,1/$00+-,583+/,****320.*-,)(&,,3(5)))*8300.+,*))-3),8;,+$&%#'(',*(&)+,+--.72/1/-)1)&$'./('$$&++(%&()-',(,-.31,)($)%&,+*,+'$--1+720*2+.1--/2,++*'&,*-,/.-,010.'*-2=393**%&'%%**,&'%&#()((('%%%&''/3*-)1/$(*..**,*+*,**&$%#%+(###'&(+.$%$&*-5,++3)/+**'(-(+.'+.*.'''#&''*((('&#$%#%&)&('),,$%%%%))43-/+29;7,+**2/.,+)(')+*0-*'''23/7:84,.'*%&)%+0,-/+0-)2.('+*&**())++2.-,+**).//,/+.2,.*,)(+./..6+*%%&/+)0+1012.,((),+55,,/.7--,/<+(,*&')('&'695%)%*+,,/653496,*$$&(''()(*'+'&+&(2*1*.+*',(,+,%('(.)%)++-4*.0),=7,'.-,0,1%-),&%&$#**(+('&))3**+*0217('/$&'((+,.)+-((%&''**()$%%'#&%&*'*+,.*(%&(-2'&**1/..'%$##(%&%()(%$%)()'%/++)**+.,)551-41(-**'*+-.%/18//-+(.,'-6,*&&%&'(%,*-*23337-(%#$&%%$%%&&-)''$**+11-1/,,,/1/02+,*'((%$$&&$%((&(&'')),+644.312*$$--(%$'-&##&(&#&((*$&%&&&'%%''((*&'$$*.*('+/,'+(&&$&*#%$###
