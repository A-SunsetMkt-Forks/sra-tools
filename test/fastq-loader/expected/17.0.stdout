       BASE_COUNT: 1696
   BIO_BASE_COUNT: 1696
   CMP_BASE_COUNT: 1696
CMP_LINKAGE_GROUP: 
     COLOR_MATRIX: 0, 1, 2, 3, 4, 1, 0, 3, 2, 4, 2, 3, 0, 1, 4, 3, 2, 1, 0, 4, 4, 4, 4, 4, 4
           CSREAD: 0130021230213123200032132310000003103000122022102013221110102023110220122300302012001021030233103112023222023133320233222102201111210020001300011111110102102102210121211222001220131210111002100310113123013212101220203021100232202020033123120010133110213323010002330221100222032102231201320332130102030300301020032132013013213111022313231230213020312133012311311100020331133000111003222131211222111022212311303020221313201312331112103010223011132220103303332012320120030010012121023313131101301101101201221302212032013022013102332102210302210122210100130333301320210302220233021321212310030212210113013231022030211300112001013320310201122000002003210001020300002020213000011332021312100130121312011021312112202233030012300330200202233101003011011301001020132010022031102131313001230131321332312133020022220022011131212001012022103132030300001003132013212312002030010123321222011202203001310202122302110122101121213200210000201233232313003120010211300122211232013000201132321320230102320320132013100322133033110020323220021222303322212001012303102320133313011020211123222100102021033301102212022110022312002200331102101101030131010313011203323010021030223223230100210202331222211221323021110030121120132220000311102311111111002122300300300033033103020020012231013311120120202110312201030023231203223300020211303333110032210132200120110021230012312111013100030103112312321201203013011133232202001312303302100322111113200210021132330320232110102102231323033012122330203232223301231202132203221203312113213300022321102211323301321310030011000023230002120120013312003301100000002102123211030332302301202313011013123231200123001232213200013233222001320013200310003320133321321010132001320023322020221333
           CS_KEY: T
        CS_NATIVE: false
   FIXED_SPOT_LEN: 1696
    LINKAGE_GROUP: 
      MAX_SPOT_ID: 1
      MIN_SPOT_ID: 1
             NAME: 1
         PLATFORM: SRA_PLATFORM_PACBIO_SMRT
          QUALITY: 2, 1, 1, 1, 2, 2, 2, 2, 2, 2, 1, 1, 2, 2, 1, 2, 2, 2, 4, 2, 2, 1, 1, 2, 1, 2, 2, 2, 2, 1, 2, 3, 4, 1, 3, 4, 6, 10, 17, 15, 4, 3, 6, 7, 2, 11, 8, 10, 13, 10, 4, 4, 1, 7, 4, 6, 9, 12, 15, 13, 14, 10, 9, 4, 4, 5, 5, 3, 8, 9, 6, 4, 4, 10, 19, 17, 9, 13, 5, 11, 8, 18, 27, 15, 19, 25, 7, 9, 9, 10, 9, 9, 7, 7, 5, 20, 5, 6, 4, 5, 8, 7, 5, 14, 19, 22, 19, 15, 13, 8, 7, 7, 4, 7, 8, 6, 5, 8, 9, 3, 5, 5, 15, 7, 13, 20, 15, 17, 16, 9, 10, 11, 7, 5, 3, 10, 16, 13, 10, 9, 21, 21, 16, 13, 12, 11, 9, 11, 10, 14, 11, 12, 7, 10, 21, 10, 8, 10, 10, 6, 9, 11, 10, 9, 9, 8, 17, 11, 9, 8, 7, 8, 13, 17, 10, 11, 16, 16, 13, 13, 12, 4, 7, 9, 9, 10, 14, 14, 8, 12, 10, 13, 10, 5, 7, 6, 5, 6, 5, 5, 6, 7, 11, 6, 3, 12, 6, 5, 7, 13, 6, 10, 8, 15, 8, 16, 9, 12, 10, 10, 10, 18, 19, 11, 7, 4, 7, 6, 6, 5, 9, 15, 12, 9, 10, 15, 18, 21, 14, 28, 29, 24, 15, 21, 15, 13, 8, 8, 7, 8, 7, 7, 6, 7, 3, 10, 19, 8, 10, 12, 7, 4, 7, 8, 10, 4, 6, 3, 6, 15, 18, 3, 4, 9, 12, 8, 14, 9, 12, 3, 7, 7, 6, 6, 8, 8, 18, 26, 21, 15, 13, 8, 7, 10, 12, 11, 10, 13, 12, 11, 11, 13, 19, 10, 16, 14, 12, 13, 14, 21, 9, 8, 5, 12, 12, 16, 12, 10, 16, 13, 14, 20, 23, 23, 20, 14, 12, 16, 17, 13, 16, 17, 14, 7, 8, 10, 9, 15, 8, 14, 7, 5, 4, 5, 9, 10, 11, 11, 10, 11, 8, 9, 13, 8, 9, 12, 12, 9, 7, 4, 2, 5, 15, 21, 18, 5, 7, 7, 10, 10, 15, 18, 11, 15, 10, 10, 7, 9, 12, 22, 17, 5, 5, 2, 4, 1, 1, 5, 9, 9, 11, 10, 10, 10, 11, 12, 22, 12, 12, 8, 13, 12, 10, 9, 8, 8, 10, 13, 16, 8, 9, 8, 4, 4, 3, 4, 8, 4, 10, 6, 11, 7, 6, 9, 6, 9, 9, 9, 13, 10, 11, 16, 24, 15, 17, 14, 10, 12, 15, 16, 6, 6, 5, 5, 4, 8, 9, 9, 15, 10, 4, 4, 18, 5, 7, 9, 20, 16, 18, 12, 9, 15, 16, 12, 11, 17, 13, 13, 20, 13, 14, 10, 11, 7, 4, 5, 13, 10, 11, 8, 9, 6, 7, 9, 9, 9, 8, 11, 9, 5, 6, 9, 7, 8, 13, 5, 5, 15, 12, 14, 11, 8, 8, 9, 7, 12, 14, 8, 12, 18, 25, 18, 17, 15, 8, 12, 6, 6, 4, 14, 20, 16, 14, 16, 14, 12, 10, 10, 8, 11, 13, 12, 14, 22, 13, 12, 9, 5, 14, 10, 16, 9, 7, 8, 6, 7, 9, 6, 7, 5, 6, 5, 10, 6, 6, 4, 7, 10, 12, 8, 11, 12, 4, 6, 15, 7, 10, 3, 6, 4, 8, 4, 5, 5, 8, 3, 3, 6, 3, 4, 4, 5, 8, 8, 13, 5, 4, 13, 12, 8, 4, 4, 4, 2, 3, 11, 11, 11, 13, 6, 6, 3, 3, 5, 6, 3, 7, 5, 7, 12, 7, 5, 7, 9, 19, 17, 15, 8, 11, 21, 14, 8, 12, 7, 2, 4, 7, 6, 6, 6, 9, 6, 9, 2, 2, 6, 6, 13, 21, 24, 18, 13, 2, 4, 7, 5, 7, 10, 9, 13, 12, 8, 10, 13, 8, 10, 16, 16, 9, 8, 7, 11, 6, 10, 7, 6, 7, 16, 14, 9, 3, 3, 7, 9, 10, 17, 17, 15, 16, 14, 13, 10, 17, 8, 13, 13, 20, 19, 10, 8, 7, 7, 7, 15, 9, 8, 4, 4, 6, 3, 3, 6, 6, 6, 7, 9, 15, 15, 13, 12, 12, 9, 18, 10, 23, 19, 20, 12, 12, 5, 10, 10, 12, 14, 8, 18, 7, 8, 8, 3, 5, 2, 3, 8, 8, 8, 14, 10, 14, 11, 10, 18, 12, 10, 10, 12, 13, 13, 14, 15, 10, 9, 12, 9, 7, 6, 10, 12, 10, 19, 13, 13, 22, 23, 12, 11, 11, 15, 13, 12, 13, 16, 18, 16, 16, 15, 11, 11, 9, 10, 12, 10, 8, 16, 12, 12, 8, 3, 10, 15, 14, 15, 16, 13, 15, 8, 9, 17, 21, 11, 5, 5, 6, 9, 10, 9, 4, 8, 10, 5, 8, 10, 10, 11, 14, 8, 9, 12, 11, 16, 10, 9, 19, 12, 12, 16, 19, 21, 20, 19, 10, 10, 12, 12, 19, 7, 11, 11, 11, 13, 7, 17, 11, 8, 8, 10, 15, 16, 14, 15, 11, 11, 10, 9, 8, 10, 22, 11, 19, 13, 15, 6, 14, 14, 9, 5, 10, 8, 8, 7, 6, 12, 8, 13, 16, 9, 4, 8, 9, 3, 6, 13, 9, 20, 6, 5, 5, 3, 5, 14, 16, 8, 9, 8, 13, 12, 12, 12, 12, 13, 10, 10, 5, 8, 9, 11, 7, 9, 7, 11, 16, 14, 3, 15, 15, 10, 12, 11, 20, 23, 18, 10, 14, 11, 9, 9, 9, 9, 18, 17, 15, 13, 9, 12, 11, 8, 7, 5, 11, 11, 18, 7, 20, 8, 8, 8, 9, 23, 18, 15, 15, 13, 10, 11, 9, 8, 8, 12, 18, 8, 11, 23, 26, 11, 10, 3, 5, 4, 2, 6, 7, 6, 11, 9, 7, 5, 8, 10, 11, 10, 12, 12, 13, 22, 17, 14, 16, 14, 12, 8, 16, 8, 5, 3, 6, 13, 14, 7, 6, 3, 3, 5, 10, 10, 7, 4, 5, 7, 8, 12, 6, 11, 7, 11, 12, 13, 18, 16, 11, 8, 7, 3, 8, 4, 5, 11, 10, 9, 11, 10, 6, 3, 12, 12, 16, 10, 22, 17, 15, 9, 17, 10, 13, 16, 12, 12, 14, 17, 11, 10, 10, 9, 6, 5, 11, 9, 12, 11, 14, 13, 12, 11, 15, 16, 15, 13, 6, 9, 12, 17, 28, 18, 24, 18, 9, 9, 4, 5, 6, 4, 4, 9, 9, 11, 5, 6, 4, 5, 2, 7, 8, 7, 7, 7, 6, 4, 4, 4, 5, 6, 6, 14, 18, 9, 12, 8, 16, 14, 3, 7, 9, 13, 13, 9, 9, 11, 9, 10, 9, 11, 9, 9, 5, 3, 4, 2, 4, 10, 7, 2, 2, 2, 6, 5, 7, 10, 13, 3, 4, 3, 5, 9, 12, 20, 11, 10, 10, 18, 8, 14, 10, 9, 9, 6, 7, 12, 7, 10, 13, 6, 10, 13, 9, 13, 6, 6, 6, 2, 5, 6, 6, 9, 7, 7, 7, 6, 5, 2, 3, 4, 2, 4, 5, 8, 5, 7, 6, 8, 11, 11, 3, 4, 4, 4, 4, 8, 8, 19, 18, 12, 14, 10, 17, 24, 26, 22, 11, 10, 9, 9, 17, 14, 13, 11, 10, 8, 7, 6, 8, 10, 9, 15, 12, 9, 6, 6, 6, 17, 18, 14, 22, 25, 23, 19, 11, 13, 6, 9, 4, 5, 8, 4, 10, 15, 11, 12, 14, 10, 15, 12, 8, 17, 13, 7, 6, 10, 9, 5, 9, 9, 7, 8, 8, 10, 10, 17, 13, 12, 11, 10, 9, 9, 8, 13, 14, 14, 11, 14, 10, 13, 17, 11, 13, 9, 11, 8, 7, 10, 13, 14, 13, 13, 21, 10, 9, 4, 4, 5, 14, 10, 8, 15, 10, 16, 15, 16, 17, 13, 11, 7, 7, 8, 11, 10, 20, 20, 11, 11, 14, 13, 22, 12, 12, 11, 14, 27, 10, 7, 11, 9, 5, 6, 8, 7, 6, 5, 6, 21, 24, 20, 4, 8, 4, 9, 10, 11, 11, 14, 21, 20, 18, 19, 24, 21, 11, 9, 3, 3, 5, 7, 6, 6, 7, 8, 7, 9, 6, 10, 6, 5, 10, 5, 7, 17, 9, 16, 9, 13, 10, 9, 6, 11, 7, 11, 10, 11, 4, 7, 6, 7, 13, 8, 4, 8, 10, 10, 12, 19, 9, 13, 15, 8, 11, 28, 22, 11, 6, 13, 12, 11, 15, 11, 16, 4, 12, 8, 11, 5, 4, 5, 3, 2, 9, 9, 7, 10, 7, 6, 5, 8, 8, 18, 9, 9, 10, 9, 15, 17, 16, 22, 7, 6, 14, 3, 5, 6, 7, 7, 10, 11, 13, 8, 10, 12, 7, 7, 4, 5, 6, 6, 9, 9, 7, 8, 3, 4, 4, 6, 2, 5, 4, 5, 9, 6, 9, 10, 11, 13, 9, 7, 4, 5, 7, 12, 17, 6, 5, 9, 9, 16, 14, 13, 13, 6, 4, 3, 2, 2, 7, 4, 5, 4, 7, 8, 7, 4, 3, 4, 8, 7, 8, 6, 4, 14, 10, 10, 8, 9, 9, 10, 13, 11, 8, 20, 20, 16, 12, 19, 16, 7, 12, 9, 9, 6, 9, 10, 12, 13, 4, 14, 16, 23, 14, 14, 12, 10, 7, 13, 11, 6, 12, 21, 11, 9, 5, 5, 4, 5, 6, 7, 4, 11, 9, 12, 9, 17, 18, 18, 18, 22, 12, 7, 4, 2, 3, 5, 4, 4, 3, 4, 4, 5, 5, 12, 8, 6, 6, 3, 9, 9, 10, 16, 16, 12, 16, 14, 11, 11, 11, 14, 16, 14, 15, 17, 10, 11, 9, 6, 7, 7, 4, 3, 3, 5, 5, 3, 4, 7, 7, 5, 7, 5, 6, 6, 8, 8, 11, 10, 21, 19, 19, 13, 18, 16, 17, 9, 3, 3, 12, 12, 7, 4, 3, 6, 12, 5, 2, 2, 5, 7, 5, 2, 5, 7, 7, 9, 3, 5, 4, 5, 5, 5, 6, 4, 4, 6, 6, 7, 7, 9, 5, 6, 3, 3, 9, 13, 9, 7, 6, 10, 14, 11, 6, 10, 7, 5, 5, 3, 5, 9, 2, 4, 3, 2, 2, 2
        RD_FILTER: SRA_READ_FILTER_PASS
             READ: UGCCCUGAUUCAUGAUCCCCGACGAUGGGGGGGCAAUUUUGAGGAGUUCCAUCUGUGGUUCCUACAAGAACUCGGGCCUUGAAACCUGGCCUAUGGCACUUCGAGAAGCAUAUCCUAUCUCAAGAACACAGUUUCCCCAUUUUGUGUGUGGUUCAAGUUCUGGUCAGUGAGAAACUCCAUGACCACAAAGUUUACCACGUCGGUAGUCAACUCCUUAAGUGGGAUCUUCCUUUAUGAUGAAACCAUACAAGUAUCGGUUUUCGCCUCACCCUCUUAGUUCUACUUGCUUAUCAUUGGAAUUAAAUUGGAAAUCAUCCAUUGCUGCACAAGAUGCUACUAAGUAAGGCAGUAUUGAUGUACACCCCUUAUGUAUUUUGUGGGCUCUGCAGUGAGACACCUCUGAUGUAAUUCCUCAUGCUUGCAGCGUGUCAAUUGGAGCCACAUCUCCAAUAAUAUCCAGCUUGAAAUUUGGGUCAGUUCGCAUGCACCAUUGUUGUUGAACUCAUUCUGAAUCCAUUCUUGCAAGCGACCUCAAUUCUGGUCUCAACCCAUUAUAUUGCUUCAAUUCUCCUAUUCAUCAGUCGUUUAAGUCUGGUGCCAUCGUUCUUAAGUGCCCACUUUGGUAUCCGUUCCACUCCCCCCUUUAGUUUUGGAAUUUUUCCUUCAUUUUUGUAUCCUGCAGUUUGCCAGUACUUGUUCAUGACAGAAGAUAAUUUGAUUUAUUCCCUUCUAUGGUUUAACAACAUUGGGUUCCAUCCAAAGAAUGUUCAUGCAUUUGAUUGCAUCAUAGCAGUAUUCCCUCUCCCUCCACAUGACUUUGGUCCUCAAUGCUUAAUUUUUGGGCAUCCAUCAGCAGGGAAUUUGGUCGCUGAGAACAGGAGGCCCAUGGAAGUCUAAGUGGUCUGGUGACUGCUUUCAAAAAGGUCGCUAGCAUUUACUUUGGACAUUUGAGACAGCUUGCCCCUUGUAGCUGCUUCGGUUCGAAUCCAUCCAUGGGCUCAUAAUACAAAGGCUAGAAAGUCUCGGCGAGACUUUGGUCGGCAAGCUUGCGCAUUGUUCCUGUGAUCUCAAACCUUCAAUAUUGUUCUGAAGACAAAGAUGAAAGAAAUACAAGUUGUUGGCCAUGGUUACGGUGAAUAGCCAAAGUUAAGAUCUAGCCAAAGUUCCUAUGAGAGUGAGUAGCCUGUGGGCCAGUGAACGAGAAAAAUGUGGAUGUGUGUGUUUCAGAUUUAAAUUUUAUUAUGGCCUUUCCCAGAUGGUAUGUGAACUUCCUGUUACUCCAAUUUCGAUGAAUCUAUUUUCCUGUAAUAUACAAAUCUGGUAGAAACUUGUUUCAGCCCAGCAGUGUUGCAAAAUUGGCACUACUAGUCCAGGCCAUUGUGCGAUCUUCCCAUGAUUAUUCAAAUCUGUGUGCUUUCAAAGUGCUAUUAGGAUCACCAAGUUCUACGAUUAUUGACUCGCCUUAGCUCUAUUGAUGAAGUAGAAUCUGAAUACUGUAGUAUUUUCUAGUGGAGUGCUAUUGCUGCAAAUUUGUUUUUCGAUUUUCAGGUCCCAUACUUUAUUGUUUUUUUUCAAGUCGACAAUUAUCGGAUUGAAGCAUUGUUGCAGCUACUUUGAUUUGAUCUGCUUUUGCUAUCUCCCAUCCCAUCCCGUUUUAUCCAUAUCAUCAACCAUCCCAUCCCUAUCUUCCUCAUAU
      READ_FILTER: SRA_READ_FILTER_PASS
         READ_LEN: 1696
         READ_SEG: [0, 1696]
       READ_START: 0
        READ_TYPE: SRA_READ_TYPE_BIOLOGICAL
       SIGNAL_LEN: 0
       SPOT_COUNT: 1
       SPOT_GROUP: 
          SPOT_ID: 1
         SPOT_LEN: 1696
         TRIM_LEN: 1696
       TRIM_START: 0

